doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -73,9 +73,14 @@ Zigbee
     It is meant to be used only for debugging purposes.
   * Documentation page about :ref:`ug_zigbee_commissioning`.
   * Experimental support for Zigbee Green Power Combo Basic functionality.
+  * Zigbee device definition for each Zigbee sample and application.
+  * Identify handler in the :ref:`Zigbee Light Switch sample <zigbee_light_switch_sample>`.
```

Comment:
I'd say this should go under Samples / Zigbee samples, further down, line 312ff.

Reply:
Moved!